src/einsteinpy/tests/test_bodies/test_body.py

```diff
@@ -0,0 +1,89 @@
+from unittest import mock
+
+import astropy.units as u
+import numpy as np
+import pytest
+
+from einsteinpy.bodies import Body
+from einsteinpy.plotting import ScatterGeodesicPlotter
+
+
+@pytest.fixture()
+def dummy_data():
```

Comment:
Good. A better name for the fixture probably?

Reply:
Will think of something good.